apps/web/src/components/commons/Button.tsx

```diff
@@ -27,6 +28,7 @@ const getButtonPadding = (size: ButtonSize): string => {
 
 export default function Button({
   size = "medium",
+  variant = "solid",
```

Comment:
lets not use default variant, this will mess up with other component style
<img width="600" alt="image" src="https://github.com/BirthdayResearch/metascan/assets/53080940/ecd1f93e-a2df-4add-adcf-8ee4aa1a25d2">